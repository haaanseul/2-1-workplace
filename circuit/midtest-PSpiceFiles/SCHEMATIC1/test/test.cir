** Profile: "SCHEMATIC1-test"  [ C:\Users\82104\Desktop\workplace\circuit\midtest-PSpiceFiles\SCHEMATIC1\test.sim ] 

** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
* From [PSPICE NETLIST] section of C:\Users\82104\AppData\Roaming\SPB_Data\cdssetup\OrCAD_PSpice\17.2.0\PSpice.ini file:
.lib "nomd.lib" 

*Analysis directives: 
.DC LIN V_V1 0 3.3v 0.1v 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 


.END
