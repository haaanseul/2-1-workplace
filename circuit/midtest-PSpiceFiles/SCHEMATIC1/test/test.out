
**** 04/24/23 18:44:27 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-test"  [ C:\Users\82104\Desktop\workplace\circuit\midtest-PSpiceFiles\SCHEMATIC1\test.sim ] 


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
* From [PSPICE NETLIST] section of C:\Users\82104\AppData\Roaming\SPB_Data\cdssetup\OrCAD_PSpice\17.2.0\PSpice.ini file:
.lib "nomd.lib" 

*Analysis directives: 
.DC LIN V_V1 0 3.3v 0.1v 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 



**** INCLUDING SCHEMATIC1.net ****
* source MIDTEST
V_V1         N00121 0 0Vdc
R_R1         N00121 N00144  10k TC=0,0 
Q_Q1         N00148 N00144 0 Q2N3904
R_R2         N00148 VCC  470 TC=0,0 
V_V2         VCC 0 12Vdc

**** RESUMING test.cir ****
.END

INFO(ORPSIM-15423): Unable to find index file nomd.ind for library file nomd.lib.

INFO(ORPSIM-15422): Making new index file nomd.ind for library file nomd.lib.

Index has 34104 entries from 205 file(s).

**** 04/24/23 18:44:27 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-test"  [ C:\Users\82104\Desktop\workplace\circuit\midtest-PSpiceFiles\SCHEMATIC1\test.sim ] 


 ****     BJT MODEL PARAMETERS


******************************************************************************




               Q2N3904         
               NPN             
       LEVEL    1            
          IS    6.734000E-15 
          BF  416.4          
          NF    1            
         VAF   74.03         
         IKF     .06678      
         ISE    6.734000E-15 
          NE    1.259        
          BR     .7371       
          NR    1            
         ISS    0            
          RB   10            
          RE    0            
          RC    1            
         CJE    4.493000E-12 
         VJE     .75         
         MJE     .2593       
         CJC    3.638000E-12 
         VJC     .75         
         MJC     .3085       
        XCJC    1            
         CJS    0            
         VJS     .75         
          TF  301.200000E-12 
         XTF    2            
         VTF    4            
         ITF     .4          
          TR  239.500000E-09 
         XTB    1.5          
          KF    0            
          AF    1            
          CN    2.42         
           D     .87         



          JOB CONCLUDED

**** 04/24/23 18:44:27 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-test"  [ C:\Users\82104\Desktop\workplace\circuit\midtest-PSpiceFiles\SCHEMATIC1\test.sim ] 


 ****     JOB STATISTICS SUMMARY


******************************************************************************



  Total job time (using Solver 1)   =        3.39
